src/server/database/Updater/UpdateFetcher.cpp

```diff
@@ -67,11 +67,11 @@ void UpdateFetcher::FillFileListRecursively(Path const& path, LocaleFileStorage&
             LocaleFileEntry const entry = { itr->path(), state };
 
             // Check for doubled filenames
-            // Since elements are only compared through their filenames this is ok
+            // Because elements are only compared by their filenames, this is ok
             if (storage.find(entry) != storage.end())
             {
-                TC_LOG_FATAL("sql.updates", "Duplicated filename occurred \"%s\", since updates are ordered " \
-                    "through its filename every name needs to be unique!", itr->path().generic_string().c_str());
+                TC_LOG_FATAL("sql.updates", "Duplicate filename \"%s\" occurred. Because updates are ordered " \
+                    "by its filename, every name needs to be unique!", itr->path().generic_string().c_str());
```

Comment:
Sure, I will make that change. When it comes to grammar or wording in text strings and comments, nitpickings or suggestions can hardly be too small. ;)